### DISCUSSION TITLE
How to stop wandblogger uploading the checkpoint?

### DISCUSSION THREAD

**Discussion by stdoo** [Lightning-AI/pytorch-lightning#7177] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I want the checkpoint and the logs stay in the same place while only the logs are uploaded to wandb server.

**Comment by justusschock** [comment]
cc @borisdayma

**Comment by borisdayma** [comment, marked as answer]
Yes, it will be fixed with #6231

**Reply by borisdayma** [reply]
Yes it should now be fixed (on master branch only for now)

**Reply by stdoo** [reply]
> Yes it should now be fixed (on master branch only for now)

I have upgrade the pytorch-lightning to 1.3.3 and wandb to 0.10.31 and set the log_model flag in WandbLogger to False, while the checkpoint is still uploaded to wandb server. How to reproduce this?

**Reply by borisdayma** [reply]
pip install git+https://github.com/pytorchlightning/pytorch-lightning.git@master

**Reply by stdoo** [reply]
Thanks a lot!
